README.md

```diff
@@ -203,7 +204,7 @@ The delay in milliseconds after which the blame info will be displayed.
 
 Note that this doesn't affect the performance of the plugin.
 
-Default: `0`
+Default: `250`
```

Comment:
I'm not sure that we should change this default value as it is a breaking change. Maybe include a note instead in the performance section to change this to `250` for better results?

Reply:
In fact, the reasons for the above issues are all caused by not turning on debounce by default. I think it should be prioritized to ensure that users use normal settings by default.

Reply:
Well, the only problem is that it changes the default behavior. I'm ok with this change but we just need to be aware of this

Reply:
I think just setting the commit message to [break change] after squashing this PR is ok

Reply:
this is something i was also not sure about when i initially checked the pr, but didn't have the time to reply. i do think it could indeed be the right thing to do, but still unsure, unfortunately.

Reply:
> this is something i was also not sure about when i initially checked the pr, but didn't have the time to reply. i do think it could indeed be the right thing to do, but still unsure, unfortunately.

We can squash the entire PR and set the commit message is  `break change`, so that people who update the plugin can notice this change.

Reply:
I have been using the pr branch to work after the submit. It currently performs well on windows and linux, and there has been no lag.

Reply:
This is also the solution for other plugins to modify the default settings. In fact, this setting change is insensitive to most users.
It only affects those users who have modified `delay`. I don’t think anyone will move the cursor and look at the commit message of quick switching at the same time.

Reply:
Yup, that makes sense :) 
I'd rather not use squash, though